packages/http-signature-utils/src/utils/signatures.ts

```diff
@@ -1,10 +1,9 @@
-import { sign, KeyLike, BinaryLike } from 'crypto'
-import {
-  httpis as httpsig,
-  Algorithm,
-  RequestLike,
-  Signer
-} from 'http-message-signatures'
+import { type KeyLike } from 'crypto'
+import { httpbis, createSigner, Request } from 'http-message-signatures'
+
+export interface RequestLike extends Request {
+  body?: string
```

Comment:
> Does this mean body is unnecessary in general?

Chose to keep the `body` property for the `RequestLike` interface because we determine in `createSignatureHeaders` whether to include `content-*` components and pass only the necessary properties when calling `signMessage`.

https://github.com/interledger/open-payments/blob/2f83578738869cf7741a34d176d9f9714772e616/packages/http-signature-utils/src/utils/signatures.ts#L34-L46